### PR TITLE
Admin Users redesign

### PR DESCRIPTION
**Pull request by marcosjr182**
This PR add a couple pages under `admin_user` routes redesign, which were missing from #62.

- [x] redesign pages under `admin_user` routes, which were missing from #62.
- [x] add button on `edit_team` so the users can access those pages

### Before
oldedit
oldindex

### After
newedit
newindex

### CODE REVIEW

File: app/views/admin/users/index.html.erb [hunk 1]
```diff
@@ -1,40 +1,60 @@
...
+      <div class="panel-body">
+        <table class="table table-condensed table-striped table-hover admin-users-table">
+          <tbody>
+            <% @users.order(:name).each do |user| %>
+              <tr>
+                <td><%= user.name %></td>
+                <td><%= user.username %></td>
+                <td><%= user.initials %></td>
+                <td><%= user.role %></td>
+                <td><%= link_to user.email, "mailto:#{user.email}" %></td>
```
Comment (talyssonoc) [inline app/views/admin/users/index.html.erb]: You can use [mail_to](http://apidock.com/rails/ActionView/Helpers/UrlHelper/mail_to) here

**Comment by akitaonrails** [comment]
How do you access this page? I am not finding the menu item.

**Comment by marcosjr182** [comment]
I have added a new button on `edit_team`. What do you think? 
btnteammembers

**Comment by marclerodrigues** [comment]
LGTM

File: app/views/admin/users/index.html.erb [hunk 2]
```diff
@@ -1,40 +1,56 @@
...
+            <% @users.order(:name).each do |user| %>
+              <tr>
+                <td><%= user.name %></td>
+                <td><%= user.username %></td>
+                <td><%= user.initials %></td>
+                <td><%= user.role %></td>
+                <td><%= mail_to user.email %></td>
+                <td>
+                  <div class="users_options btn-group pull-right">
+                    <% if policy(user).update? %>
```
Comment (rafaelrpbelo) [inline app/views/admin/users/index.html.erb]: Talvez seja interessante extrair essa lógica para um presenter.

💅

Comment (marcosjr182) [inline app/views/admin/users/index.html.erb]: Agreed! =) I am will be moving it to a presenter on a separate PR to refactor a couple views.
